src/media/media.style.js

```diff
@@ -4,9 +4,19 @@
  * @param hasRadius Does the media rounded or not ?
  * @param color The media color
  */
-export default ({ size = 40, hasRadius = false, color = '#efefef' }) => ({
-  height: size,
-  width: size,
-  borderRadius: hasRadius ? size / 2 : 3,
-  backgroundColor: color,
-});
+export default ({
+  size = 40,
+  hasRadius = false,
+  color = '#efefef',
+  width,
+  height,
+  borderRadius,
+}) => {
+  const defaultRadius = hasRadius ? size / 2 : 3;
+  return {
+    height: height || size,
+    width: width || size,
+    borderRadius: typeof borderRadius === 'number' ? borderRadius : defaultRadius,
```

Comment:
Technically I could have done
```js
borderRadius: typeof borderRadius === 'number' ? borderRadius : hasRadius ? size / 2 : 3
```
to keep the implicit return, but nested inline if statements are quite hard to read.